cpp/source/deal.II/geometry.h

```diff
@@ -44,18 +44,27 @@ class Geometry
 
   virtual ~Geometry() = default;
 
+  void set_anode_boundary_id(dealii::types::boundary_id const boundary_id)
+  {
+    _anode_boundary_id = boundary_id;
+  }
+
```

Comment:
is it possible to have a device with multiple ``boundary_ids`` tagged as anode/cathode?